### PR TITLE
Fix `.gitignore` entry for `/.spago`.

### PR DESCRIPTION
**Pull request by nsaunders**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,6 @@
 /documentation/.psci*
 
 /documentation/html/test.js
-./spago
+/.spago
```
Comment (JordanMartinez) [inline .gitignore:18]: Can't this be just `.spago/`?

Comment (nsaunders) [reply .gitignore:18]: Indeed, I think `.spago/` or even just `.spago` would work. With the leading `/`, I believe a file/directory named `.spago` located in a subdirectory _would not_ be ignored, so maybe `.spago` would be the best pattern to use... What do you think @JordanMartinez ?

Comment (JordanMartinez) [reply .gitignore:18]: I'd go with `.spago/`

Comment (thomashoneyman) [reply .gitignore:18]: I believe there is a template .gitignore file in the PureScript-contrib governance repo that can be used here

Comment (nsaunders) [reply .gitignore:18]: Fair enough @thomashoneyman... I'll open a new PR.

**Pull request closed without merging**